sources/Adapters/picoTracker/filesystem/picoTrackerFileSystem.h

```diff
@@ -7,6 +7,17 @@
 #include <string.h>
 #include <vector>
 
+enum PathIndexTypes {
+  ParentDirIndex,
+  DirIndex,
+  FileIndex,
+};
+
+struct PathIndex {
+  unsigned short index;
```

Comment:
Use fixed width integer types here (uint16_t?)

Reply:
Good call will do.

Reply:
oops! sorry about that @democloid I forgot to push up the commit! I'll fix that up in next PR.